modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/paths/CP2NavigationMenu.path

```diff
@@ -95,6 +95,36 @@
 	<td>//a[contains(@class, 'font-weight-bold')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>SEARCH_FILTER</td>
+	<td>//div[contains(@class,'bg-neutral-1')]//div[@class="input-group-item"]//input[@placeholder="Search"]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>SEARCH_RESULT</td>
+	<td>//p[contains(@class, 'font-weight-semi-bold m-0 mt-3 text-paragraph-sm') and contains(.,'result')]</td>
+	<td></td>
+</tr>
```

Comment:
@styller1 the way this locator is written, with this class inside the contains, it is not error prone. The likelihood of it to change is high. Change "font-weight-semi-bold m-0 mt-3 text-paragraph-sm" to only "text-paragraph-sm" and check if it works